### PR TITLE
Bruno tests for internal rights delegation

### PR DESCRIPTION
**Pull request by ivartryti**
## Description

Added new Bruno tests for rightsinternal (and removed unnecessary console.log lines):
- person to person delegation
- delegating person revokes delegation
- receiving person revokes delegation

- person to org delegation
- delegating person revokes delegation
- receiving org revokes delegation

- org to person delegation
- delegation org revokes delegation
- receiving person revokes delegation

- org to org delegation
- delegating org revokes delegation
- receiving org revokes delegation

## Developer/Reviewer Checklist
- [ ] **Your** code builds clean without any errors or warnings
- [ ] No changes to config/appsettings or environment variables created in separate linked PR
- [ ] Manual testing done (required)
- [ ] Relevant integration test added (required for functional changes)
- [ ] Relevant automated test added (required for functional changes)
- [ ] All integration tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: test/Bruno/Altinn.AccessManagement/Automatic Test Collection/RightsInternal/o2o org2org delegation successful.bru [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  auth: none
+}
+
+headers {
+  Authorization: {{apiKey}}
+}
+
+vars:pre-request {
+  auth_tokenType: Personal
+  auth_userId: 20000490
```
Comment (jonkjetiloye) [inline test/Bruno/Altinn.AccessManagement/Automatic Test Collection/RightsInternal/o2o org2org delegation successful.bru:19]: Kan vurdere om default verdier på variablene burde cleares siden disse skrives over av TestData i script

File: test/Bruno/Altinn.AccessManagement/Automatic Test Collection/RightsInternal/o2o receiving org no longer has delegation.bru [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+get {
+  url: {{baseUrl}}/accessmanagement/api/v1/internal/{{party}}/rights/delegation/received
+  body: json
+  auth: none
+}
+
+headers {
+  Authorization: {{apiKey}}
```
Comment (jonkjetiloye) [inline test/Bruno/Altinn.AccessManagement/Automatic Test Collection/RightsInternal/o2o receiving org no longer has delegation.bru:14]: apiKey? Blir ikke denne styrt på collection nivå av {{bearerToken}}

**Review by jonkjetiloye — APPROVED** [review]
Nice! 👍 
Konge at du har dekket alle o2o, o2p, p20 og p2p scenarioene.

Eneste jeg evt. savner er om noen av de scenarioene hadde delegert Ressurs og Altinn 2 tjeneste. Nå delegerer vel alle en Altinn App.
Uansett en mye bedre dekning enn jeg var forberedt på!

File: test/Bruno/Altinn.AccessManagement/Automatic Test Collection/RightsInternal/o2o org2org delegation successful.bru (hunk 1 above)
Comment (ivartryti) [reply test/Bruno/Altinn.AccessManagement/Automatic Test Collection/RightsInternal/o2o org2org delegation successful.bru]: clearet variablene som hentes fra testdatafilene men beholder de som ikke finnes i testdatafilene

**Pull request merged**